### DISCUSSION TITLE
Speeding up VVenC by increasing refreshsec

### DISCUSSION THREAD

**Discussion by jungleboynx** [fraunhoferhhi/vvenc#314] [Show and tell]
This is just an observation, not an issue.
Setting -rs 4 (instead of the default -rs 1) increases encode speed from 4.5fps to 6.5fps (preset fast 1080p mode on 6C/12T CPU).
It'll also reduce the file size slightly as there's fewer I-SLICE frames.

**Comment by adamjw24** [comment]
Yes, thats true. Intra-period is usually not something you can choose, and especially not for encoding speed reasons. Unless of course, it's about you personal video library, in which case its a very valid optimization option.
